app/tests/settings.py

```diff
@@ -10,9 +10,15 @@
 from config.settings import *  # noqa: F401, F403, E402, I251
 
 SESSION_COOKIE_DOMAIN = ".testserver"
-ALLOWED_HOSTS = [SESSION_COOKIE_DOMAIN]
+ALLOWED_HOSTS = [SESSION_COOKIE_DOMAIN, ".localhost"]
```

Comment:
It seems like you're fighting the existing setting of `.testserver` here, especially when modifying the default site object in `conftest.py`.

Would it work if the `SESSION_COOKIE_DOMAIN` was `.localhost`? If so, it might be better to create a different PR to main where that is changed for all of the tests, and then you can start with something that is aligned with what you're trying to do here.

Reply:
Only setting `SESSION_COOKIE_DOMAIN` to `.localhost` without also overwriting the domain (as we do in conftest) does not work.

Reply:
So changing this for all tests would require getting rid of the `django_db_setup` fixture that sets the domain to `testserver`. Not sure what consequences that would have.

Reply:
> Only setting SESSION_COOKIE_DOMAIN to .localhost without also overwriting the domain (as we do in conftest) does not work.

Yes, I wouldn't expect that to work, it's just that you have two of them now fighting each other, one which is auto loaded and one which is used as a fixture. The idea would be to unify them and use one.

Reply:
We no longer need to fight the testerver now that we're not testing the subdomains anymore, so closing this